web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -490,12 +490,12 @@ public static function getBlockedSettings(): array {
       'blocked-patron-f-body-text' => t('Blocked patron f body text', [], ['context' => 'Blocked user']),
       'blocked-patron-e-title-text' => t('You have exceeded your fee limit', [], ['context' => 'Blocked user']),
       'blocked-patron-e-body-text' => t('You are therefore not able to borrow or reserve materials from the library', [], ['context' => 'Blocked user']),
-      'blocked-patron-w-title-text' => t('Your user is blocked', [], ['context' => 'Blocked user']),
-      'blocked-patron-w-body-text' => t('You therefore cannot reserve, borrow or renew loans. Please contact the library for further information', [], ['context' => 'Blocked user']),
+      'blocked-patron-w-title-text' => t('Your user is blocked (w)', [], ['context' => 'Blocked user']),
+      'blocked-patron-w-body-text' => t('You therefore cannot reserve, borrow or renew loans. Please contact the library for further information (w)', [], ['context' => 'Blocked user']),
       'blocked-patron-o-title-text' => t('Blocked reason O modal title', [], ['context' => 'Blocked user']),
       'blocked-patron-o-body-text' => t('Blocked patron o body text', [], ['context' => 'Blocked user']),
-      'blocked-patron-u-title-text' => t('Your user is blocked', [], ['context' => 'Blocked user']),
-      'blocked-patron-u-body-text' => t('You therefore cannot reserve, borrow or renew loans. Please contact the library for further information', [], ['context' => 'Blocked user']),
+      'blocked-patron-u-title-text' => t('Your user is blocked (u)', [], ['context' => 'Blocked user']),
+      'blocked-patron-u-body-text' => t('You therefore cannot reserve, borrow or renew loans. Please contact the library for further information (u)', [], ['context' => 'Blocked user']),
```

Comment:
Please use the same pattern as for all blocked texts to ensure consistency.

I do not understand what `d`, `s`, `f`, `w` and `u`means and I guess it is outside the scope of this PR to provide this information. Preceeding lines add the special letters in the middle of the text e.g. "Blocked patron d title text". Here it is added in a parenthesis at the end.  

Personally I perfer the last option but the most important thing is to keep it consistent. This makes it easier to editors to look up all variations of the text.

I suggest that you change the title and body texts in this PR to the existing forms e.g. "Blocked patron w title text" and "Blocked patron w body text".

Alternately change the existing texts to the form introduced in this PR e.g. "Your user is blocked (d)", "You therefore cannot reserve, borrow or renew loans. Please contact the library for further information (d)".

Reply:
I fixed this. Went with the parentheses at the end for them all.